frontend/src/CreateShipment.js

```diff
@@ -41,93 +41,120 @@ export default function CreateShipment() {
-          placeholder="eta"
-          maxLength="40"
-          required
-        />
-        <InputStyled type="submit" value="Abschicken" />
-      </FormStyled>
+      <BackgroundWrapperTop>
+        <h3>Erstelle Dein Shipment</h3>
+      </BackgroundWrapperTop>
+      <InputWrapper>
+        <FormStyled onSubmit={handleSubmit}>
+          <LabelStyled htmlFor="Bl">Bl Nr.:</LabelStyled>
+          <InputStyled
+            id="Bl"
+            name="Bl"
+            type="text"
+            value={shipmentData.Bl}
+            onChange={updateEntry}
+            placeholder="Bl Nr."
+            maxLength="40"
+            required
+          />
+
+          <LabelStyled htmlFor="Palettenanzahl">Palettenanzahl:</LabelStyled>
+          <InputStyled
+            id="Palettenanzahl"
+            name="Palettenanzahl"
+            type="number"
+            value={shipmentData.Palettenanzahl}
+            onChange={updateEntry}
+            placeholder="Palettenanzahl"
+            maxLength="40"
+            required
+          />
+
+          <LabelStyled htmlFor="Lieferant">Lieferant:</LabelStyled>
+          <InputStyled
+            id="Lieferant"
+            name="Lieferant"
+            type="text"
+            value={shipmentData.Lieferant}
+            onChange={updateEntry}
+            placeholder="Lieferant"
+            maxLength="40"
+            required
+          />
+
+          <LabelStyled htmlFor="Warenbeschreibung">
+            Warenbeschreibung:
+          </LabelStyled>
+          <InputStyled
+            id="Warenbeschreibung"
+            name="Warenbeschreibung"
+            type="text"
+            value={shipmentData.Warenbeschreibung}
+            onChange={updateEntry}
+            placeholder="Warenbeschreibung"
+            maxLength="40"
+            required
+          />
+          <DateWrapper>
+            <LabelDate htmlFor="etd">Schiffabfahrt Ursprungsland:</LabelDate>
+            <InputDate
+              id="etd"
+              name="etd"
+              type="date"
+              value={shipmentData.etd}
+              onChange={updateEntry}
+              required
+            />
+
+            <LabelDate htmlFor="eta">Schiffsankunft Hamburg:</LabelDate>
+            <InputDate
+              id="eta"
+              name="eta"
+              type="date"
+              value={shipmentData.eta}
+              onChange={updateEntry}
+              required
+            />
+          </DateWrapper>
+          <InputButton type="submit">Daten Übermitteln</InputButton>
+        </FormStyled>
+      </InputWrapper>
     </main>
   )
   function updateEntry(e) {
     setShipmentData({ ...shipmentData, [e.target.name]: e.target.value })
   }
 }
 
-const Headline = styled.h2`
-  color: #fff;
-  text-align: center;
+const BackgroundWrapperTop = styled.div`
+  width: 375px;
+  height: 359px;
+  background-image: linear-gradient(115deg, #2156e7 6%, #1345d0 93%);
+
+  h3 {
+    font-family: 'Roboto', sans-serif;
```

Comment:
Du könntest die font-family vielleicht einfach in den global styles im body oder wenn sie nur für bestimmte elemente gilt z.b für h3 festlegen, dann musst du sie nicht für die einzelnen elemente jedes mal definieren.